hk2-locator/src/test/java/org/glassfish/hk2/tests/locator/optional/NamedOptionalInjection.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2012, 2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright shouldn't be Oracle on a new file you can either use;

`Copyright (c) {year} Contributors to the Eclipse Foundation`

or 

`Copyright (c) {year} {owner}`

See https://www.eclipse.org/projects/handbook/#ip-copyright-headers

Also year should be 2020

Reply:
Hey sorry for the inconvenience :)
I changed the licence according to the link.